src/main/java/study/step4/RacingGame.java

```diff
@@ -0,0 +1,46 @@
+package study.step4;
+
+import study.step4.domain.Car;
+import study.step4.move.MoveStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+
+    private final List<Car> carList = new ArrayList<>();
+    private final MoveStrategy moveStrategy;
+
+    public RacingGame(String[] carNames, MoveStrategy moveStrategy){
+        this.moveStrategy = moveStrategy;
+        for (String carName : carNames) {
+            this.carList.add(new Car(carName));
+        }
+    }
+
+    public List<Car> getCarList() {
+        return carList;
+    }
+
+    public List<Car> getWinnerList() {
+        List<Car> winnerList = new ArrayList<>();
+        int maxPosition = -1;
+
+        for (Car car : carList) {
+            if (car.getPosition() > maxPosition) {
+                maxPosition = car.getPosition();
+                winnerList.clear();
+                winnerList.add(car);
+            } else if (car.getPosition() == maxPosition) {
+                winnerList.add(car);
+            }
+        }
+        return winnerList;
+    }
+
+    public void play() {
+        for (Car car : carList) {
+                car.move(moveStrategy);
+        }
+    }
```

Comment:
intellij를 사용하고 계신가요? 3단계에 있었던 요구사항이었는데요
단축키를 사용하면 쉽게 컨벤션을 맞출 수 있습니다!

![image](https://github.com/next-step/java-racingcar/assets/36617783/d11596b2-a0bf-46a8-b0b0-be17b8bdd08a)
```suggestion
    public void play() {
        for (Car car : carList) {
            car.move(moveStrategy);
        }
    }
```